scrapy/settings/default_settings.py

```diff
@@ -141,7 +141,7 @@
 FEED_TEMPDIR = None
 FEEDS = {}
 FEED_URI_PARAMS = None  # a function to extend uri arguments
-FEED_STORE_EMPTY = False
+FEED_STORE_EMPTY = True
```

Comment:
We need to update [the docs](https://github.com/scrapy/scrapy/blob/d60b4edd11436e61284615ec7ce89f8ac7e46d9a/docs/topics/feed-exports.rst?plain=1#L555) accordingly.

Reply:
I forgot about that.
Other than changing the default values, do I need to make any additions to the documentation with this modification? For example, mentioning that files are not exported explicitly, etc.

Reply:
To mention that files are not exported may be a good thing, yes.

I don’t think we need anything else, though. We will need to justify this default value change, explain the reasons behind it, in the release notes, but we usually handle release notes entries when preparing the release, so you do not need to worry about that part here.

Reply:
Understood. Thank you very much.